src/main/java/org/example/bookstore/exception/InvalidPriceException.java

```diff
@@ -0,0 +1,7 @@
+package org.example.bookstore.exception;
+
+public class InvalidPriceException extends RuntimeException {
```

Comment:
I do not see any usage of that exception except in the `ExceptionHandler`, so let's remove that